master/docs/conf.py

```diff
@@ -28,6 +28,7 @@
 # Add any Sphinx extension module names here, as strings. They can be extensions
 # coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
 extensions = [
+    'sphinx.ext.autodoc',
```

Comment:
When I ran "make docs", it failed inside Pillow and said that this module was not recognized. Adding this line fixed that issue.